bool    isPlaying
string  motionName
float32 motionDuration
float32 currentTime